suites/apis/googleServiceAccountTest.js

```diff
@@ -100,6 +59,57 @@ Scenario('Register Google Service Account Success @reqGoogle @first', async (fen
   fence.ask.responsesEqual(deleteRes, fence.props.resDeleteServiceAccountSuccess);
 });
 
+Scenario('Google project locking test @reqGoogle', async (fence, google) => {
```

Comment:
does this run before everything else?

Reply:
does this test take a long time to run? with retries and whatnot?

Reply:
it does not run first :(
it runs first in the `googleServiceAccountTest` suite, but from the logs it looks like `googleServiceAccountRemovalTest` runs before `googleServiceAccountTest`. but it will still help to understand what's wrong if we see it fail
it took 80 sec just now on my dev env